giskard/scanner/llm/llm_prompt_injection_detector.py

```diff
@@ -30,9 +30,13 @@ class LLMPromptInjectionDetector(Detector):
     .. [#] Leon Derczynsky, garak:  LLM vulnerability scanner, https://github.com/leondz/garak
     """
 
-    def __init__(self, threshold: float = 0.5, num_samples=100):
+    def __init__(self, threshold: float = 0.5):
         self.threshold = threshold  # default
-        self.num_samples = num_samples
```

Comment:
@rabah-khalek could you double check that this makes sense?

Reply:
Yes, `num_samples` is only actively used in https://github.com/Giskard-AI/giskard/pull/1533/. It's a nice feature to have, but not a blocker for the release.